### ISSUE TITLE
Eval / train mode switch

### ISSUE THREAD

**Issue by lironbk74** [benmyara/pytorch-examples#2]
In some of the notebooks where `batchnorm` and/or `dropout` are used, the network is not switched properly between `eval` and `train` mode.

**Comment by bainjamain** [comment]
That's right, will edit soon.
